static/js/services/delete-docs.js

```diff
@@ -61,6 +62,14 @@ var utilsFactory = require('bulk-docs-utils');
       };
 
       var bulkDeleteRemoteDocs = function (docs, eventListeners) {
+        // TODO: we're temporarily (?) killing the changes feed here for performance
+        // Having the changes feed watching and then disseminating changes to the whole
+        // page causes massive performance issues while large deletes are occurring.
+        // We need to fix this either by improving performance in this area or by
+        // radically change how we follow changes for online users
+        // https://github.com/medic/medic-webapp/issues/4327
+        Changes({die: true});
```

Comment:
So this call to `Changes` effectively kills the changes feed? If so this seems like an odd API to me, calling it with `die: true` rather than having some sort of `Changes.killWatchers()`, but if that's going to be a big refactor for a temporary change then I can see why you'd rather leave it as is.

Reply:
I don't think it would be a big refactor to do this, nice suggestion!